pkg/master/reconcilers/lease.go

```diff
@@ -119,16 +120,16 @@ func NewLeases(storage storage.Interface, baseKey string, leaseTime time.Duratio
 }
 
 type leaseEndpointReconciler struct {
-	endpointRegistry      endpoint.Registry
+	endpointStorage       *genericregistry.Store
```

Comment:
maybe this could be a rest.StandardStorage instead of *genericregistry.Store and then the registrytest could just change its signatures slightly to get the tests to compile

Reply:
sounds good